<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockTrade</class>
 <widget class="QWidget" name="StockTrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>持有股票信息</string>
  </property>
  <widget class="QTableWidget" name="HoldingTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>511</width>
     <height>471</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>股票编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>购买单价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>持有数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>购买总价</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
